Emby.Server.Implementations/Dto/DtoService.cs

```diff
@@ -1099,7 +1099,9 @@ private void AttachBasicFields(BaseItemDto dto, BaseItem item, BaseItem owner, D
 
                 Series episodeSeries = null;
 
-                if (options.ContainsField(ItemFields.SeriesPrimaryImage))
+                // this block will add the series poster for episodes without a poster
+                // TODO maybe remove the if statement entirely
+                //if (options.ContainsField(ItemFields.SeriesPrimaryImage))
                 {
```

Comment:
If we're commenting out the if, don't the braces need to go away?

Reply:
Nope C# is cool with it.